appshell/cefclient_mac.mm

```diff
@@ -334,8 +334,15 @@ - (void)windowWillEnterFullScreen:(NSNotification *)notification {
     }
     if (customTitlebar) {
         [customTitlebar setHidden:YES];
+        NSWindow *window = [notification object];
+
+        // Since we have nuked the title, we will have
+        // to set the string back as we are hiding the
+        // custom title bar.
+        [window setTitle:[customTitlebar titleString]];
```

Comment:
@nethip 
I also see that 

> OS X Deployment Target is 10.6. 

Shouldn't that be updated? Changing that causes appshell build to fail. The errors and deprecations should be fixed.
Also npm install for Brackets throws these deprecations -

> npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
> npm WARN deprecated npmconf@2.1.1: this package has been reintegrated into npm and is now out of date with respect to npm
> npm WARN deprecated graceful-fs@1.1.14: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
> npm WARN deprecated graceful-fs@3.0.8: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
> npm WARN deprecated lodash@1.0.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.